crates/common/tedge_utils/src/config.rs

```diff
@@ -43,6 +37,37 @@ impl Default for SystemSpecificCommands {
     }
 }
 
+#[derive(Deserialize, Debug, Eq, PartialEq)]
+#[serde(deny_unknown_fields)]
+pub struct LogConfig {
+    #[serde(default = "default_log_level")]
+    pub tedge_mapper: String,
+    #[serde(default = "default_log_level")]
+    pub tedge_agent: String,
+    #[serde(default = "default_log_level")]
+    pub tedge_watchdog: String,
+    #[serde(default = "default_log_level")]
+    pub c8y_log_plugin: String,
+    #[serde(default = "default_log_level")]
+    pub c8y_config_plugin: String,
+}
```

Comment:
We cannot have a struct like this one to manage service log levels. This struct would have to be updated each and every time a new service is added.

One need a `HashMap` or even simply a vector of `(String, LogLevel)` pairs.